sample-extension-plugin/src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/sampleextension/SampleJobParameter.java

```diff
@@ -110,19 +118,26 @@ public void setIndexToWatch(String indexToWatch) {
         this.indexToWatch = indexToWatch;
     }
 
+    public void setLockDurationSeconds(Long lockDurationSeconds) {
+        this.lockDurationSeconds = lockDurationSeconds;
+    }
+
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
         builder.field(NAME_FIELD, this.jobName)
-                .field(ENABLED_FILED, this.isEnabled)
-                .field(SCHEDULE_FIELD, this.schedule)
-                .field(INDEX_NAME_FIELD, this.indexToWatch);
-        if(this.enabledTime != null) {
+            .field(ENABLED_FILED, this.isEnabled)
```

Comment:
NP: ```ENABLED_FILED``` -> ```ENABLED_FIELD```

Reply:
Existing in one of the cherry-picked commits, can clean this up in other branches + here later.